common/client/node_lifecycle_test.go

```diff
@@ -837,6 +836,26 @@ func TestUnit_NodeLifecycle_unreachableLoop(t *testing.T) {
 
 		setupRPCForAliveLoop(t, rpc)
 
+		node.declareUnreachable()
+		tests.AssertEventually(t, func() bool {
+			return node.State() == nodeStateAlive
+		})
+	})
+	t.Run("on successful verification without isSyncing becomes alive", func(t *testing.T) {
```

Comment:
nit: This same description is repeated multiple times for different test cases

Reply:
Yes, but each of them is located in different testing functions and test similar transition: from unreachable to alive, from invalidChainID to alive, etc.